addons/stock_account/models/stock_valuation_layer.py

```diff
@@ -13,7 +13,6 @@ class StockValuationLayer(models.Model):
 
     _rec_name = 'product_id'
 
-    active = fields.Boolean(related='product_id.active')
```

Comment:
Shouldn't we display the Archive/Unarchive button only if the field is stored ? 

Seems better than removing the field knowing that its side effect is to remove the Archived button that shouldn't be displayed. If the field is reintroduced afterward it will bring back the issue + it could happen in a lot of other models.

Reply:
And that way, we don't need a migration script ;)

Reply:
@tivisse , I agree with you, we should display the Archive/Unarchive button on stored field only. But regrading this field change it's already removed in master in PR: https://github.com/odoo/odoo/pull/45586/files#diff-4b85c61639e6834a4fb3a4a813c6a3fdL16
And also migration script is already merged. so just need to rebase the branch.

Thanks